dax/internal/client/tubepool.go

```diff
@@ -72,6 +80,10 @@ func newTubePoolWithOptions(address string, options tubePoolOptions, connConfigD
 	if options.maxConcurrentConnAttempts <= 0 {
 		options.maxConcurrentConnAttempts = defaultTubePoolOptions.maxConcurrentConnAttempts
 	}
+	if options.minConnections < 1 {
+		options.minConnections = defaultTubePoolOptions.minConnections
+	}
+	// TODO: validation for max connections?
```

Comment:
why is not being added the max connection validation in the same way is validated the maxConcurrentConnAttempts?